portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,15 @@ function addRandomGreeting() {
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function loadingContentOfUserInput() {
+    const serverResponse  = await fetch('/hello');
+    const textdisplayed = await serverResponse.json();
```

Comment:
Also use lower camel case here and below for all variable names. See https://google.github.io/styleguide/jsguide.html#naming-local-variable-names

Reply:
Done